lua/autorun/client/cl_focus.lua

```diff
@@ -0,0 +1,26 @@
+local tabbedOutList = {}
+local Icon = Material( "icon16/application_double.png" ) -- the afk icon
+
+timer.Create( "tabNetTimmer", 1, 0, function()
+	if not system.HasFocus() then -- called when the client tabs out
+		net.Start( "tabbedOut" )
+		net.SendToServer()
+	else -- called when the client tabs back in
+		net.Start( "tabbedIn" )
+		net.SendToServer()
+	end
+
+	net.Receive( "sendData", function() -- receives the players that tabbed out

```

Comment:
You don't have to define the net.Recieve every X amount of time, you only have to call it once, you can put this outside of the timer.